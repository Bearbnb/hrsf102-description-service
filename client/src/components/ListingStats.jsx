import React from 'react';
import PropTypes from 'prop-types';

const ListingStats = ({
  guests, bedrooms, beds, baths,
}) => (
  <div>
    <div>{guests} guests {bedrooms} bedrooms {beds} beds {baths} baths</div>
  </div>
);

ListingStats.propTypes = {
  guests: PropTypes.number,
  bedrooms: PropTypes.number,
  beds: PropTypes.number,
  baths: PropTypes.number,
};

ListingStats.defaultProps = {
  guests: 0,
  bedrooms: 0,
  beds: 0,
  baths: 0,
};

export default ListingStats;
