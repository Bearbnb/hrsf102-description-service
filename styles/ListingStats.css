.container {
  width: 65%;
  margin: 20px auto 25px auto;
}