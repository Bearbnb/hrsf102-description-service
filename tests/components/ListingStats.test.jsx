import { shallow } from 'enzyme';
import React from 'react';
import toJson from 'enzyme-to-json';
import ListingStats from '../../client/src/components/ListingStats';

describe('<ListingStats />', () => {
  it('should render properly', () => {
    const wrapper = shallow(<ListingStats />);

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
