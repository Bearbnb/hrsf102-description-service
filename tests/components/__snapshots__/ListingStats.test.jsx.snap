// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ListingStats /> should render properly 1`] = `
<div
  className="container"
>
  <div
    className="statOne"
  >
    <img
      alt=""
      src="https://s3-us-west-1.amazonaws.com/airbnb-owner-photos/guests.png"
    />
    <div>
      0
       guests
    </div>
  </div>
  <div
    className="statTwo"
  >
    <img
      alt=""
      src="https://s3-us-west-1.amazonaws.com/airbnb-owner-photos/living.png"
    />
    <div>
      0
       bedrooms
    </div>
  </div>
  <div
    className="statThree"
  >
    <img
      alt=""
      className="img3"
      src="https://s3-us-west-1.amazonaws.com/airbnb-owner-photos/beds.png"
    />
    <div>
      0
       beds
    </div>
  </div>
  <div
    className="statFour"
  >
    <img
      alt=""
      src="https://s3-us-west-1.amazonaws.com/airbnb-owner-photos/bath.png"
    />
    <div>
      0
       baths
    </div>
  </div>
</div>
`;
